parlai/chat_service/utils/misc.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Miscellaneous utils for chat services.
+"""
+import importlib
+import math
+import re
+from enum import Enum
+
+THREAD_MEDIUM_SLEEP = 0.3
+
+
+def set_is_debug(is_debug):
+    global debug
+    debug = is_debug
```

Comment:
This really belongs to logging utils, it's a bit strange to set a global from a different file.

Reply:
ahh great point i'll move in a later PR